keras_nlp/models/llama/llama_attention.py

```diff
@@ -86,21 +97,30 @@ def build(self, inputs_shape):
             name="attention_softmax",
         )
 
+        self._dropout_layer = keras.layers.Dropout(
+            rate=self._dropout,
+            dtype=self.dtype_policy,
+        )
+
         self._output_dense = keras.layers.EinsumDense(
-            equation="bqm,mh->bqh",
-            output_shape=(None, self.hidden_dim),
-            kernel_initializer=clone_initializer(self.kernel_initializer),
+            equation="bquh,uhm->bqm",
```

Comment:
How did this work before? Or did it not?

Reply:
It did work since we reshaped the inputs from (B, Q, U, H) -> (B, Q, U * H) before passing them to the output dense layer. Both the ways are valid. I just used the same thing I wrote for Mistral.